index.html

```diff
@@ -3228,8 +3264,12 @@ <h2 id="OfflineAudioContext">
                 <li>Otherwise, in the case that the buffer was successfully
                 constructed, <a>begin offline rendering</a>.
                 </li>
+                <li>Append <var>promises</var> to <a>pendingPromises</a>.
```

Comment:
"Promises"? Isn't there only one for `startRendering`?

Reply:
yes, typo.